cypress/e2e/newCompanySpec.cy.js

```diff
@@ -1,112 +1,134 @@
-import { mockCompanies } from "../support/mockCompanies";
+import { mockCompanies } from "../fixtures/mockCompanies.js";
 
 describe("New Company page after logging in", () => {
   let uniqueCompanyName = `Test Company ${Date.now()}`;
 
   beforeEach(() => {
-    cy.visit("http://localhost:3000/");
-
-    cy.intercept('POST', 'http://localhost:3001/api/v1/users/2/companies', {
-      statusCode: 201,
-      body: {
-        id: 123,
-        name: uniqueCompanyName,
-        website: 'www.testcompany.com',
-        street_address: '123 Test St',
-        city: 'Test City',
-        state: 'CO',
-        zip_code: '12345',
-        notes: 'Test notes'
-      }
-    }).as('addCompany');
-    
-    cy.intercept("GET", "http://localhost:3001/api/v1/users/2/companies", {
+    // Intercept the login POST request
```

Comment:
Any particular reason you moved the login down? I don't think the company data is received until the actual page link is clicked.